client/src/components/subscription/subscriptionForm.js

```diff
@@ -124,103 +126,142 @@ class SubscriptionForm extends React.Component {
+            <div className="text-center d-none d-md-block">On which day(s) do you want to be reminded?</div>
+            <div className="col mx-auto">
+              <ReactDayPicker handleUpdate={this.handleDays} updating={days} nextDueDate={nextDueDate} frequency='yearly' />
+            </div>
           </div>
         );
       case 'monthly':
         return (
-          <div>
-            <label htmlFor="occurrence">
-              Every (?) Months:
-              <input type="number" id="occurrence" value={occurrence} onChange={(event) => this.handleChange(event, 'occurrence')} min="0" max="12" />
-            </label>
-            <div>On which day(s) do you want to be reminded?</div>
-            <ReactDayPicker handleUpdate={this.handleDays} updating={days} nextDueDate={nextDueDate} frequency='monthly' />
+          <div className="col-12 d-flex">
+            <div className="col-md-6">
+              <label className="col d-flex form-label" htmlFor="occurrence">
+                <div className="col-4 col-md-3">Every (?) Months:</div>
+                <div className="col-8 col-md-7">
+                  <input className="form-control" type="number" id="occurrence" value={occurrence} onChange={(event) => this.handleChange(event, 'occurrence')} min="0" max="12" />
+                </div>
+              </label>
+            </div>
+            <div className="col-md-6 mx-auto">
+              <div className="text-center d-none d-md-block">On which day(s) do you want to be reminded?</div>
+              <div id="dayPickerForm">
+                <ReactDayPicker handleUpdate={this.handleDays} updating={days} nextDueDate={nextDueDate} frequency='monthly' />
+              </div>
+            </div>
           </div>
         );
       case 'weekly':
         return (
-          <div>
-            <label htmlFor="occurrence">
-              How many weeks in between do you want to be reminded?
-              <select id="occurrence" placeholder="ex: Every 2 weeks" value={occurrence} onChange={(event) => this.handleChange(event, 'occurrence')}>
-                <option value="1">Every Week</option>
-                <option value="2">Every 2 Weeks</option>
-                <option value="3">Every 3 Weeks</option>
-                <option value="4">Every 4 Weeks</option>
-              </select>
-            </label>
-            <div>Day of the week:</div>
-            <div>{weeklyCheckbox}</div>
-            <ReactDayPicker updating={days} nextDueDate={nextDueDate} frequency='weekly' />
+          <div className="col-12 d-flex">
+            <div className="col-md-6">
+              <label className="col d-flex form-label" htmlFor="occurrence">
+                <div className="col-4 col-md-3">Every (?) Weeks:</div>
+                <div className="col-8 col-md-7">
+                <select className="form-control" id="occurrence" placeholder="ex: Every 2 weeks" value={occurrence} onChange={(event) => this.handleChange(event, 'occurrence')}>
```

Comment:
The dropdown doesn't look like a select dropdown and instead looks like a text input field b/c it doesn't have the little arrow on the right side. I'd suggest using the `form-select` bootstrap class for this instead of the `form-control` that you're currently using

```suggestion
                <select className="form-select" id="occurrence" placeholder="ex: Every 2 weeks" value={occurrence} onChange={(event) => this.handleChange(event, 'occurrence')}>
```